src/qobj.jl

```diff
@@ -1,7 +1,15 @@
 using Random
 
+IBMQClient.@option struct ExpOptions
+    id::String
+    header::IBMQClient.Maybe{Dict} = nothing
+    nshots::Int = 1024
+    exp_header::IBMQClient.Maybe{Vector} = nothing
```

Comment:
```suggestion
    exp_header::IBMQClient.Maybe{Vector{Any}} = nothing
```

Reply:
We can have it as a `Dict` IMO.

Reply:
either `Dict` or `Any`, but should put a type there.

Reply:
if you put a `Dict`, remember to fill the parameters as `Dict{String, Any}`